src/test/java/com/gizasystems/automationexercise/tests/verifyingSubscriptionInHomePage.java

```diff
@@ -0,0 +1,49 @@
+package com.gizasystems.automationexercise.tests;
+
+import com.gizasystems.automationexercise.pages.*;
+import com.shaft.driver.SHAFT;
+import io.qameta.allure.Epic;
+import io.qameta.allure.Feature;
+import io.qameta.allure.Story;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.annotations.BeforeClass;
+
+@Epic("Automation Exercise")
+@Feature("Home page verification")
+@Story("Verify Subscription in home page")
+
+public class verifyingSubscriptionInHomePage {
+
+    private SHAFT.GUI.WebDriver driver;
+    private SHAFT.TestData.JSON testUser;
+    private String timeStamp = String.valueOf(System.currentTimeMillis());
+
+    @Test(description = "Verify Subscription is on Home Page")
```

Comment:
Please add Description and Tms annotations (@TmsLink should have this value ("55512381")) 
![image](https://github.com/MahmoudElSharkawyGS/AutomationExercisePractice/assets/46620469/d0361c60-ac59-4abe-91e8-5873b4096587)